src/runtime/go.mod

```diff
@@ -132,11 +135,5 @@ require (
 	tags.cncf.io/container-device-interface/specs-go v0.7.0 // indirect
 )
 
-replace (
-	github.com/go-openapi/swag => github.com/go-openapi/swag v0.21.1
-	github.com/opencontainers/runc => github.com/opencontainers/runc v1.1.9
-	github.com/stretchr/testify => github.com/stretchr/testify v1.8.0
-	github.com/uber-go/atomic => go.uber.org/atomic v1.5.1
```

Comment:
This one directive may have been introduce on purpose? Not only pins version but changes actual source location. Maybe a quick check in history to see whether this was introduced on purpose?

Reply:
This import isn't used in the code